phpstan.neon

```diff
@@ -34,4 +34,7 @@ parameters:
 	doctrine:
 		objectManagerLoader: tests/object-manager.php
 		allowNullablePropertyForRequiredField: true
+	symfony:
+		containerXmlPath: var/cache/dev/AppKernelDevDebugContainer.xml
+		consoleApplicationLoader: app/console-application.php
```

Comment:
@mabumusa1 is this even necessary? We had a call with @fedys and it seems that PHPSTAN runs successfully even without this line.

Reply:
@escopecz the first line is needed according to the [docs here](https://github.com/phpstan/phpstan-symfony#configuration)

For the 2nd line, it is need to analyze the commands as [documented here  ](https://github.com/phpstan/phpstan-symfony#analysis-of-symfony-console-commands)

So I think they are needed